relay-sampling/src/lib.rs

```diff
@@ -205,26 +213,19 @@ mod tests {
         }
     }
 
-    fn mocked_dynamic_sampling_context(
-        transaction: &str,
-        release: &str,
-        environment: &str,
-        user_segment: &str,
-        user_id: &str,
-        replay_id: Option<Uuid>,
-    ) -> DynamicSamplingContext {
+    fn mocked_dsc(environment: &str, user_segment: &str, release: &str) -> DynamicSamplingContext {
```

Comment:
removed the fields that weren't changing in any of the tests. 

shorter names to ensure calling them in one line for easier readability